liquidator/liquidator.js

```diff
@@ -133,8 +133,8 @@ class Liquidator {
     // The `price` is a BN that is used to determine if a position is liquidatable. The higher the
     // `price` value, the more collateral that the position is required to have to be correctly collateralized.
     // Therefore, we add a buffer by deriving scaledPrice = price * (1 - crThreshold)
-    const scaledPrice = this.fromWei(
-      price.mul(this.toBN(this.toWei("1")).sub(this.toBN(this.toWei(this.crThreshold.toString()))))
+    const scaledPrice = Math.round(
+      this.fromWei(price.mul(this.toBN(this.toWei("1")).sub(this.toBN(this.toWei(this.crThreshold.toString())))))
```

Comment:
Unrelated to this line -- I'm assuming the liquidator and disputer are not expected to wrap and unwrap their WETH. They'll basically just need pre-wrapped WETH for their collateral inventory, right?

That was my assumption, but just wanted to be sure.

Reply:
ye, this is my assumption as well. same as adding any other collateral currency to the bot.

Reply:
My assumption as well. We might want to add a module to automatically do this, or we can just take on more work on the front-end and just send the bots wrapped ETH

Reply:
+1 -- I think loading them up with WETH ahead of time is good for now. No need for premature optimization.